source/Calamari/Commands/ReleasePackageLockCommand.cs

```diff
@@ -31,10 +26,27 @@ public ReleasePackageLockCommand(IVariables variables, IManagePackageUse package
             this.variables = variables;
             this.log = log;
             this.packageJournal = packageJournal;
+            timeBeforePackageExpiration = GetTimeBeforeExpiration(variables.Get(KnownVariables.Calamari.PackageRetentionLockExpiration));
```

Comment:
Let's just pass the variables to the `GetTimeBeforeExpiration(..)` method, so the constructor doesn't need to know anything about what variable is actually needed.

Reply:
Done and done.